config/environments/production.rb

```diff
@@ -20,4 +20,16 @@
   end
 
   config.active_record.dump_schema_after_migration = false
+
+  config.email_from = 'game_panel@gmail.com'
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address:              'smtp.gmail.com',
+    port:                 587,
+    domain:               'example.com',
+    user_name:            'email',
```

Comment:
We need use ENV here
`{ user_name: ENV['email_user_name'], password: ENV['email_password'] }